modules/ignite-cli-2/src/test/java/org/apache/ignite/cli/commands/cliconfig/CliConfigSetSubCommandTest.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.ignite.cli.commands.cliconfig;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import jakarta.inject.Inject;
+import org.apache.ignite.cli.commands.CliCommandTestBase;
+import org.apache.ignite.cli.config.Config;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class CliConfigSetSubCommandTest extends CliCommandTestBase {
+
+    @Inject
+    Config config;
+
+    @BeforeEach
+    void setUp() {
+        setUp(CliConfigSetSubCommand.class);
+    }
+
+    @Test
+    @DisplayName("Need at least one parameter")
+    void noKey() {
+        // When executed without arguments
+        execute();
+
+        // Then
+        assertThat(err.toString()).contains("Missing required parameter");
+        // And
+        assertThat(out.toString()).isEmpty();
+    }
+
+    @Test
+    @DisplayName("Missing value")
+    void singleKey() {
+        // When executed with key
+        execute("ignite.cluster-url");
+
+        // Then
+        assertThat(err.toString()).contains("should be in KEY=VALUE format");
```

Comment:
Why don't check the whole err output?

Reply:
For brevity - whole output is 3 lines, I don't see any benefit in checking it - will add the property name though.